README.md

```diff
@@ -122,6 +122,28 @@ $ touch WORKSPACE # declare that to be our workspace root
 $ bazel build gemmlowp:all
 ```
 
+### Building using cmake
```

Comment:
Can we keep the CMake stuff separate from this CL?
We had yet to decide
 - how we'd communicate the support status of the CMake build, since we don't have the resources to maintain it or even ensure we don't break it.
 - at least give a try at coming to some agreement among the multiple users who have proposed CMake support, see at least https://github.com/google/gemmlowp/pull/97 by @adevress

Reply:
done, If you want them back in the future, let me know.